src/renderer/families/polkadot/SimpleOperationFlowModal/steps/StepInfo.js

```diff
@@ -0,0 +1,87 @@
+// @flow
+import invariant from "invariant";
+import React from "react";
+import { Trans } from "react-i18next";
+
+import type { StepProps } from "../types";
+
+import TrackPage from "~/renderer/analytics/TrackPage";
+import Box from "~/renderer/components/Box";
+import Button from "~/renderer/components/Button";
+import ErrorBanner from "~/renderer/components/ErrorBanner";
+import InfoBox from "~/renderer/components/InfoBox";
+import Text from "~/renderer/components/Text";
+import AccountFooter from "~/renderer/modals/Send/AccountFooter";
+
+export default function StepInfo({
+  account,
+  parentAccount,
+  onUpdateTransaction,
+  transaction,
+  status,
+  bridgePending,
+  warning,
+  error,
+  t,
+  mode,
+}: StepProps) {
+  invariant(
+    account && account.polkadotResources && transaction,
+    "account and transaction required",
+  );
+
+  return (
+    <Box flow={1}>
+      <TrackPage category="SimpleOperationFlow Flow" name="Step 1" />
+      {warning && !error ? <ErrorBanner error={warning} warning /> : null}
+      {error ? <ErrorBanner error={error} /> : null}
+      <Box px={5} py={2}>
+        {/* <Box horizontal justifyContent="center" mb={4}>
```

Comment:
Extra comments ?

Reply:
I kept it supposing Final Design would include a per-mode illustration.

But yes, if there is no illustration to integrate, i should be removed completely.